addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -925,10 +929,10 @@ export class Rtc extends Record {
         this.state.serverInfo = serverInfo;
         this.state.channel.rtcSessions = rtcSessions;
         this.selfSession = this.store.RtcSession.get(sessionId);
-        this.state.iceServers = iceServers || DEFAULT_ICE_SERVERS;
+        this.iceServers = iceServers || DEFAULT_ICE_SERVERS;
```

Comment:
Since `iceServers` is a self-adjusted computed field, no need to `|| DEFAULT_ICE_SERVERS`